README.md

```diff
@@ -8,8 +8,8 @@ Prerequisites (Unix only):
     * GCC 4.2 or newer
     * G++ 4.2 or newer
     * Python 2.6 or 2.7
-    * GNU Make 3.81 or newer
-    * libexecinfo (FreeBSD and OpenBSD only)
+    * Windows and Linux and MacOS
```

Comment:
Seems a little silly to add this under "Prerequisites (Unix only)". And why remove `make`?